hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationMonitor.java

```diff
@@ -103,32 +105,28 @@
     @Probe
     private final long invocationScanPeriodMillis = SECONDS.toMillis(1);
 
-    //todo: we need to get rid of the nodeEngine dependency
-    InvocationMonitor(NodeEngineImpl nodeEngine,
```

Comment:
When having many arguments, I prefer the original approach. It makes it a lot easier to read which parameters are passed.